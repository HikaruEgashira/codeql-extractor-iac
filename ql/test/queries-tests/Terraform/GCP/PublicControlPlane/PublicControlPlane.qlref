security/Terraform/GCP/PublicControlPlane.ql