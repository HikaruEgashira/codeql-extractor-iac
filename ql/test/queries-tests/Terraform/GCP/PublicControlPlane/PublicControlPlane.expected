| gke.hcl:13:5:15:5 | cidr_blocks | No Pod Security Policy Defined |
